code/index.html

```diff
@@ -3,28 +3,231 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylsheet" href="./normalize.css" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+    />
+    <link rel="icon" type="img/png" href="./img/flower.png">
```

Comment:
This is a really pretty detail and a huge plus for using a separate image folder! 👏